### ISSUE TITLE
TypeError during render initialization (thus unable to Render shots)

### ISSUE THREAD

**Issue by sgodanas** [ubisoft/shotmanager#23]
**Infos on the context**
- Add-on version: 1.7.10
- Blender version, from installer or from zip: 3.1 (Stable)
- Blender was launched in Admin mode? YES
- OS:  Windows 10
- Bug was also on previous versions (I don't know / No / Yes, seen on version x.x): NO

**Describe the bug**
Unable to render shots because of a TypeError

**To Reproduce**
1. Setup shots in a take.
2. Click Render all shots.

**Expected behavior**
All shots must be rendered

**Screenshots**

<img width="809" alt="Screenshot 2022-03-16 143735" src="https://user-images.githubusercontent.com/73957119/158758177-b3837170-ca36-4839-86a5-e39b5f8219a1.png">

**Additional context**

FROM: https://wiki.blender.org/wiki/Reference/Release_Notes/3.1/Python_API

```
Python has been upgraded from version 3.9 to 3.10, there are some changes that may impact script authors.

    Python 3.10 no longer implicitly converts floats to int's ([issue linked](https://bugs.python.org/issue37999)). This means functions that previously accepted float typed values will raise a type error.
    Floating point arguments must now be explicitly converted to integers (see example commits ([rBAbb62f10]
(http://developer.blender.org/rBAbb62f10715a871d7069d2b2c74b2efc97c3c350c), [rBA7476c1a](http://developer.blender.org/rBA7476c1ac248472c1442b3fb1cb3e0a79747e48f6)).

```

**Comment by werwack** [comment]
Indeed it was a Python 3.10 issue.
Thanks for the link. I saw it but didn't know where the conversion issue could appear.
I fixed all those I found but there may have some others somewhere :S We will see over time...

**Comment by sgodanas** [comment]
Great, thank you!

**Comment by werwack** [comment]
Hi @sgodanas ,
This bug has also been fixed in v1.7.16. Let me know if you still face the issue.
Thank you

**Comment by sgodanas** [comment]
Hi @werwack 

That sounds great. I will start testing today and let you know.

Thank you!
Satish.
